src/main/java/core/basesyntax/service/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class ReaderServiceImpl implements ReaderService {
+    @Override
+    public List<String> read(String fileName) {
+        try {
+            return Files.readAllLines(Path.of(fileName));
+        } catch (IOException | NullPointerException e) {
```

Comment:
It’s  not a really good idea to catch exceptions from different groups in one block